lib/server.js

```diff
@@ -71,3 +87,65 @@ function notFoundHandler (req, res, next) {
 
   return next(notFoundError)
 }
+
+/**
+ * Expand RAML 1.0 types.
+ *
+ * @param  {Object}   raml
+ * @param  {Object}
+ */
+function expandTypes (raml) {
+  var ctx = _createTypesContext(raml.types)
+  if (!ctx) {
+    return raml
+  }
+
+  function _expand (data, ctx) {
+    if (!(data instanceof Object)) {
+      return data
+    }
+
+    for (var key in data) {
+      var val = data[key]
+
+      if (val instanceof Array) {
+        data[key] = []
+        val.forEach(function (el) {
+          data[key].push(_expand(el, ctx))
+        })
+        continue
+      }
+
+      if (val instanceof Object) {
+        data[key] = _expand(val, ctx)
+        continue
+      }
+    }
+
+    var isExpandable = data.type &&
+      data.type.length === 1 &&
+      NON_EXPANDABLE_TYPES.indexOf(data.type[0]) === -1
```

Comment:
seems like an [issue](https://github.com/raml-org/raml-parser-toolbelt/issues/39) in `datatype-expansion`